share/gpodder/extensions/subscription_stats.py

```diff
@@ -0,0 +1,185 @@
+
+import gi  # isort:skip
+gi.require_version('Gtk', '3.0')  # isort:skip
+from gi.repository import Gtk, Pango  # isort:skip
+
+_ = gpodder.gettext
+
+__title__ = _('Subscription Statistics')
+__description__ = _('Show publishing statistics for subscriptions.')
+__only_for__ = 'gtk'
+__doc__ = 'https://gpodder.github.io/docs/extensions/subscription_stats.html'
+__authors__ = 'Brand Huntsman <http://qzx.com/mail/>'
+
+
+class gPodderExtension:
+    def __init__(self, container):
+        self.container = container
+
+    def on_ui_object_available(self, name, ui_object):
+        if name == 'gpodder-gtk':
+            self.gpodder = ui_object
+
+    def on_create_menu(self):
+        # extras menu
+        return [(_("Subscription Statistics"), self.open_dialog)]
+
+    def add_page(self, notebook, category, channels):
+        scrolled = Gtk.ScrolledWindow()
+        scrolled.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC)
+
+        store = Gtk.ListStore(str, str, str, int)
+        for average, name, edate, paused in channels:
+            last = strftime('%x', localtime(edate))
+            store.append([
+                ('%.1f' % round(average, 1)) if average > 0 else '?',
+                ('‖ ' if paused else '') + name, last, edate,
+            ])
+
+        tree = Gtk.TreeView(model=store)
+        scrolled.add(tree)
+
+        dayscell = Gtk.CellRendererText()
+        dayscell.set_property('xalign', 1)
+        dayscell.set_property('alignment', Pango.Alignment.RIGHT)
+        dayscolumn = Gtk.TreeViewColumn(_('Days'))
+        dayscolumn.set_sort_column_id(0)
+        dayscolumn.pack_start(dayscell, True)
+        dayscolumn.add_attribute(dayscell, 'text', 0)
+        tree.append_column(dayscolumn)
+
+        channelcell = Gtk.CellRendererText()
+        channelcell.set_property('xalign', 0)
+        channelcell.set_property('alignment', Pango.Alignment.LEFT)
+        channelcell.set_property('ellipsize', Pango.EllipsizeMode.END)
+        channelcolumn = Gtk.TreeViewColumn(_('Podcast'))
+        channelcolumn.set_sort_column_id(1)
+        channelcolumn.pack_start(channelcell, True)
+        channelcolumn.add_attribute(channelcell, 'text', 1)
+        channelcolumn.set_expand(True)
+        tree.append_column(channelcolumn)
+
+        lastcell = Gtk.CellRendererText()
+        lastcell.set_property('xalign', 0)
+        lastcell.set_property('alignment', Pango.Alignment.LEFT)
+        lastcolumn = Gtk.TreeViewColumn(_('Last Updated'))
+        lastcolumn.set_sort_column_id(3)
+        lastcolumn.pack_start(lastcell, True)
+        lastcolumn.add_attribute(lastcell, 'text', 2)
+        tree.append_column(lastcolumn)
+
+        edatecell = Gtk.CellRendererText()
+        edatecolumn = Gtk.TreeViewColumn()
+        edatecolumn.add_attribute(edatecell, 'text', 2)
+        edatecolumn.set_visible(False)
+        tree.append_column(edatecolumn)
```

Comment:
This CellRenderer and column are not necessary, adding them to the TreeView without being set visible leads to Gtk errors. Sorting works fine also without this.

Reply:
You are right, removed